icetea_lib/GenericProcess.py

```diff
@@ -124,6 +125,8 @@ def _read_fd(file_descr):
             # Store the remainded, its either '' if last char was '\n'
             # or remaining buffer before line end
             stream_desc.buf = split[-1]
+            return len(line)
```

Comment:
Actually this might still lose bytes if there is more than 1024*1024 bytes in buffer and lines 115 or 117 result in empty line, not sure what the buffer should contain for that to happen though...